tex/complex-ana/quintic.tex

```diff
@@ -0,0 +1,187 @@
+But this means that $R$ has been tricked into changing the root it outputs, contradiction!
+
+The bigger picture here is that we were able to find an operation that fixes $R$ while changing the order 
+of the roots in $S_5$.
+
+\section{Step 2: Nested Roots}
+
+Once we add $\sqrt[n]{}$ back to the picture, this idea no doesnlonger works out of the box.
+
+\begin{example}[Quadratic Formula]
+    If you've done any competition math, you know that for a polynomial 
+    $P(z) = az^2 + bz + c = (z-z_1)(z - z_2)$, it follows that the two branches of
+    \[
+        \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}
+    \]
+    give $z_1$ and $z_2$.
+
+    So why can't swapping $z_1$ and $z_2$ yield a contradiction here?
+    It's because while all the coefficients end up in the starting position,
+    the liftings of how $\sqrt{b^2 - 4ac}$ travels may not.
+\end{example}
+
+\begin{exercise}
+    Consider the polynomial $z^2 - 1$. Then smoothily swap the roots along
+    $(z - e^{it})(z + e^{it})$. See that the two roots given by the quadratic formula
+    also swap position.
+\end{exercise}
+
+Let's now consider the next simplest case of the $n$th root of a rational function
+$\sqrt[n]{R}$, and try to fix it with a nontrivial permutation of the roots.
+
+Swapping the roots $z_1$ and $z_2$, we keep $R$ the same, but $R$'s path $\alpha$ around 
+the origin may have accumulated some change in phase $2\pi a$. If we were to unswap 
+$z_1$ and $z_2$ in the same manner, we'd undo the change in phase, but we'd also be back
+to doing nothing.
+
+However, while changes in phase are \emph{abelian}, while permutations are not. Let's consider
+another operation of swapping the roots $z_2$ and $z_3$. Taking a commutator of the two 
+operations, we keep all the phases the same, but end up with a permutation 
+$(12)(23)(12)^{-1}(23)^{-1}$.
+
+If we mark the second operation's path with $\beta$, this corresponds to 
+$\alpha\beta\alpha^{-1}\beta^{-1}$.
+
+\begin{center}
+	\begin{asy}
+		unitsize(0.8cm);
+		size(7cm);
+		cplane();
+		pair A = Drawing((1.5,0));
+		dot("$R$", A, dir(-45));
+		draw(A..(0,2.5)..MP("\alpha", (-1.5,2), dir(90))..(-1.2,-1.3)..(1,1.5)..(1.8,1.1)..A, red, EndArrow);
+		draw(A..(0.7,0.5)..MP("\beta", (0.6,0.3), dir(90))..A, blue, EndArrow);
+	\end{asy}
+\end{center}
+
+
+\begin{exercise}
+    Show that this permutation operation is nontrivial.
+\end{exercise}
+
+We now have better tools: We have permutations in $S_5$ that fix the 
+$n$th roots of rational functions, and their compositions under 
+$+, -, \times, \div$.
+
+How do we handle the nested radicals now?
+
+\begin{example}[Cubic Formula]
+    The cubic formula contains a nasty term
+    \[
+        \sqrt[3]{\frac{2b^3 - 9abc + 27a^2d + \sqrt{(2b^3 - 9abc + 27a^d)^2 - 4(b^3 - 3ac)^3}}{2}}.
+    \]
+    Here, we've taken multiple roots.
+\end{example}
+
+With our new tools, we can again consider their \emph{commutators}, which 
```

Comment:
I can roughly understand the commutator part (after squinting for a bit what it's meant by Abelian), but it's not clear how the generalization is done.

Reply:
I'll try to reword this, but essentially we have operations that fix radicals of rational functions, so we can again take their commutator and the phase cancels out. Maybe I should mention explicitly that these are commutators of commutators.

There's also some exposition at https://web.williams.edu/Mathematics/lg5/ArnoldQuintic.pdf that might make more sense.

Reply:
I went and rephrased this as well.